README.md

```diff
@@ -153,113 +154,119 @@ We know that we want to yield each element of the array successively to a block
 ```ruby
 def hello_t(array)
   i = 0
+
   while i < array.length
     i = i + 1
   end
 end
 ```
 
-##### `while` Loop Review
+##### `while` loop review
 
-In the code above, we set a counter variable, `i`, equal to `0`. We start our `while` loop and tell it to execute the code in between the `while` and `end` keywords as long as `i` is less than the length of the array. Inside our `while` loop, we increment the value of our `i` variable. 
+In the code above, we set a counter variable, `i`, equal to `0`. We start our `while` loop and tell it to execute the code in between the `while` and `end` keywords as long as `i` is *less* than the length of the array. Inside our `while` loop, we increment the value of `i`.
 
-Now, we need to tell our method to actually `yield` each member of the array, as we loop over it, to the block we will call our `#hello_t` method with. 
+Now, as we iterate through the array, we need to `yield` each member of the array to the block that we'll pass to our `#hello_t` method.
 
-#### Step 3: Yielding to the Block
+#### Step 3: Yielding to the block
 
-The first time through our `while` loop, `i` is set equal to zero. The second time through the loop, `i` is set equal to `1`, and so on. This will go until until `i` is equal to the last index number of our array. 
+The first time through our `while` loop, `i` is equal to `0`. The second time through the loop, `i` is `1`, and so on. This will continue until `i` is equal to the index number of the last item in our array.
 
-So, inside each step of the `while` loop, `i` equals a given index number of our array. We can use this information to yield each successive value stored in our array, to our block. 
+So, during each step of the `while` loop, `i` equals a given index number of our array. We can use this information to yield each successive value stored in the array to the passed-in block:
 
 ```ruby
 def hello_t(array)
   i = 0
+
   while i < array.length
     yield array[i]
-		i = i + 1
+    i = i + 1
   end
 end
 ```
 
-Here, we use the `[]`, bracket, method to grab the value of each successive index element as we proceed through our `while` loop, and `yield` it to a block. Now we're ready to call our method:
+Here, we use the bracket (`[]`) method to grab the value of each successive index element as we proceed through our `while` loop, `yield`ing each to a block. Now we're ready to call our method:
 
-#### Step 4: Calling Our Method
+#### Step 4: Calling our method
 
-Let's call our method with an argument of the following array:
+Let's invoke our method, passing in the following array as an argument:
```

Comment:
We should have a discussion around consistent terminology. Do we want to use invoke or call?